components/divider/divider.tsx

```diff
@@ -1,5 +1,13 @@
 import styles from './divider.module.scss';
 
-const Divider = () => <hr className={styles.hr} />;
+interface Props {
+  color: 'dark' | 'light';
```

Comment:
can we make 'light' the default so then we don't have to change the existing ones?